services/hockeyapp/hockeyapp.tester.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const createServiceTester = require('../create-service-tester')
+
+const { isVPlusDottedVersionAtLeastOne } = require('../test-validators')
+
+const t = createServiceTester()
+module.exports = t
+
+t.create('Invalid AppToken or AppId')
+  .get('hockeyapp/v/1234/1234.json')
+  .expectJSONTypes({ name: 'hockeyapp', value: 'invalid' })
```

Comment:
If we're just comparing to an object literal, we use `expectJSON()` in preference to `expectJSONTypes()` - it gives us a more meaningful error message if it fails :)